### PR TITLE
HPCC-32050 -HPCC Portuguese language Update 9.6

### PR DESCRIPTION
**Pull request by vfumene**
## Type of change:
- [x ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x ] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by g-pan** [comment]
review ongoing - builds ok - checking content - approval pending.

**Comment by g-pan** [comment]
in the case of the 2 conflicts in both cases the later or existing one is correct and this branch is introducing a regression. 
Both cases the existing should remain.

File: docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml [hunk 1]
```diff
@@ -168,158 +190,271 @@ PLEASE NOTE: Elastic Search declares PVC(s) which might require explicit manual
...
+        <para>Uma vez habilitado e em execução, você pode explorar e consultar
+        os logs de componentes do HPCC Systems a partir da interface de
+        usuário do Kibana. Os padrões de índice do Kibana são necessários para
+        explorar dados do Elastic Search a partir da interface de usuário do
+        Kibana. Para mais informações sobre como usar a interface
+        Elastic-Kibana, por favor, consulte a documentação
+        correspondente:</para>
 
-      <para><ulink url="???">https://www.elastic.co/</ulink></para>
+        <para><ulink url="???">https://www.elastic.co/</ulink></para>
```
Comment (g-pan) [inline docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml:298]: <para><ulink url="???">https://www.elastic.co/</ulink></para> 
should be 
<para><ulink url="https://www.elastic.co/">https://www.elastic.co/</ulink></para>

File: docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml [hunk 2]
```diff
@@ -168,158 +190,271 @@ PLEASE NOTE: Elastic Search declares PVC(s) which might require explicit manual
...
 
-      <para>e</para>
+        <para>e</para>
 
-      <para><ulink
-      url="???">https://www.elastic.co/elastic-stack/</ulink></para>
+        <para><ulink
+        url="???">https://www.elastic.co/elastic-stack/</ulink></para>
+      </sect3>
```
Comment (g-pan) [inline docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml:304]: <para><ulink
        url="???">https://www.elastic.co/elastic-stack/</ulink></para>
should be 
<para><ulink
        url="https://www.elastic.co/elastic-stack/">https://www.elastic.co/elastic-stack/</ulink></para>

File: docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml (hunk 1 above)
Comment (g-pan) [reply docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml:298]: just replace the ??? with the URL - as this format has expanded my text

File: docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 3]
```diff
@@ -969,122 +1005,637 @@
...
-        taints entra em jogo.</para>
-
-        <para>Para obter mais informações e exemplos de nossos Taints,
-        Tolerations e Placements, consulte nossa documentação do
-        desenvolvedor:</para>
+    <sect2 id="ENV_values_yaml">
+      <title>Valores do Ambiente</title>
 
-        <para><ulink
-        url="???">https://github.com/hpcc-systems/HPCC-Platform/blob/master/helm/hpcc/docs/placements.md</ulink></para>
```
Comment (g-pan) [inline docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:999]: please replace ??? with URL https://github.com/hpcc-systems/HPCC-Platform/blob/master/helm/hpcc/docs/placements.md

File: docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 4]
```diff
@@ -969,122 +1005,637 @@
...
+        da VM. Se um job grande Thor vier – então o nível de taints repele
+        todos os pods que tentam ser agendados em um nó que não atende aos
+        requisitos.</para>
+
+        <para>Para mais informações e exemplos de nossos Taints, Tolerations e
+        Placements, por favor, consulte nossa documentação de
+        desenvolvedor:</para>
+
+        <para><ulink
+        url="???">https://github.com/hpcc-systems/HPCC-Platform/blob/master/helm/hpcc/docs/placements.md</ulink></para>
```
Comment (g-pan) [inline docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1404]: please replace ??? with URL https://github.com/hpcc-systems/HPCC-Platform/blob/master/helm/hpcc/docs/placements.md

**Review by g-pan — COMMENTED** [review]
see comments below 
A few regressions being introduced which were addressed recently. 
We could let them go and they would get picked up in the next round of translations, or we can resolve the conflict now and have a nice clean merge

**Comment by JamesDeFabia** [comment]
@vfumene Please let @g-pan  know when you have made the requested changes and pushed.

**Review by g-pan — APPROVED** [review]
approved

**Comment by vfumene** [comment]
@ghalliday Done, Gary. Please, let me know if any additional steps are required.

**Comment by g-pan** [comment]
Still seeing conflicts. Unable to resolve as I don't have write access. 
The ConfigureValues.xml conflict looks to be a case of both changes need to be kept - both the link in Candidate-9.6 and the Port_lang_update need to keep. 

There are 2 conflicts in the ContainerLogging.xml - the first is trivial (whitespace in the XML) either way is fine. 
The second conflict seems like the Port_Lang_update branch needs to keep.

**Comment by g-pan** [comment]
only @vfumene can resolve this. Here are the suggested steps: 
Use this technique to resolve a CONFLICT (in this Pull Request):

From Git Command Line: 
(go to your branch portuguese_lang_update_)
Issue command: 
        $ git rebase 

Should return "unable to "Auto-Merge"  CONFLICT (Content) error: <_message_> "

Open offending files (use any text editor) : _ConfigureValues.xml_ and _ContainerLogging.xml_
Edit files to remove conflict markers (<<<<<, =====, and >>>>>) and ensure required changes exist
(the last comment above) as you require.  
Save the files 
Then issue : 
    $ git add <filename>.xml 
Add the modified files back into the branch  

Issue command: 
     $ git rebase --continue  

This may open a commit msg file - edit that file - if necessary  - close and should return to command prompt. 

Issue command: 
    $ git push -f origin <branchname> 

After push completes Verify changes here in GitHub

**Comment by vfumene** [comment]
Guys, files updated. Please, check if it is ok.

**Comment by g-pan** [comment]
conflict looks resolved. 

Approved to merge.

**Pull request merged**